api/cas-server-core-api-ticket/src/main/java/org/apereo/cas/ticket/TicketDefinition.java

```diff
@@ -15,6 +15,14 @@
  * @since 5.1.0
  */
 public interface TicketDefinition extends Ordered {
+
+    /**
+     * Gets entity name.
+     *
+     * @return the entity name
+     */
+    String getTicketEntityName();
```

Comment:
I don't think this is actually needed at the API level; it's something very specific to a module and I believe the JDBC ticket registry and others do this by stuffing things into the Properties object. You may want to look into doing the same.

Reply:
Made more sense to me here as it's a property of the ticket and was pulled using a static method previously.